protocol/contracts/zaps/TempleZaps.sol

```diff
@@ -0,0 +1,504 @@
+pragma solidity ^0.8.4;
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./ZapBase.sol";
+import "./libs/Swap.sol";
+import "./interfaces/IFaith.sol";
+import "./interfaces/ITempleStableRouter.sol";
+import "./interfaces/IGenericZaps.sol";
+import "./interfaces/IVaultProxy.sol";
+import "./interfaces/IVault.sol";
+
+
+contract TempleZaps is ZapBase {
+  using SafeERC20 for IERC20;
+
+  address private constant FRAX = 0x853d955aCEf822Db058eb8505911ED77F175b99e;
```

Comment:
is this used? Should be immuatable and passed via ctor to help with testnet

Reply:
not used, will remove.
@scooprinder re testnet, given the nature of real-time dex data (from 0x) for the contracts and liquidity pools to function as expected, do you still think testnet should be on the table? thoughts?

Reply:
It's going to be really hard to maintain using testnet for STAX - eg for GMX they don't really have a maintained testnet that I can see.

Reply:
yes I agree with you @frontier159 , testing real-time data on testets is a hustle. hence why i ignored testnets until now.